ChangeLog.d/ecdsa-verify-fixes.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+   * Fix ECDSA verification, where it was failing to validate the public key
+     for signatures where R == 1 and S == 1. This bug meant that it was
+     possible to verify such signatures with an invalid public key, in some
+     cases.
```

Comment:
We should credit cryptofuzz for finding the issue and list it as https://github.com/Mbed-TLS/mbedtls/issues/4420.

I've verified manually that
```
Non-regression for #4420
depends_on:MBEDTLS_ECP_DP_SECP256K1_ENABLED
ecdsa_verify:MBEDTLS_ECP_DP_SECP256K1:"b7c52588d95c3b9aa25b0403f1eef75702e84bb7597aabe663b82f6f04ef2777":"4eef2a826f1e6d13a4dde4e54800ed6a012fcf2100ee64d53c616221eae5763a":"b7c52588d95c3b9aa25b0403f1eef75702e84bb7597aabe663b82f6f04ef2777":"b7c52588d95c3b9aa25b0403f1eef75702e84bb7597aabe663b82f6f04ef2777":"fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141":MBEDTLS_ERR_ECP_INVALID_KEY
```
now passes, but fails without the library patch in this pull request. I don't see a need to add this exact test case in addition to the ones already present.